paltax/source_models.py

```diff
@@ -293,16 +294,20 @@ def modify_cosmology_params(
         Returns:
             Modified cosmology parameters.
         """
-        npz_file = np.load(self.cosmos_path)
-        images = npz_file['images']
+
+        hdf5_file = h5py.File(self.cosmos_path, 'r')
+        images = hdf5_file['galaxy_images'][:][:][:]
         cosmology_params['cosmos_n_images'] = len(images)
-        redshifts = npz_file['redshifts']
-        pixel_sizes = npz_file['pixel_sizes']
+        redshifts = hdf5_file['redshifts'][:]
```

Comment:
.value here and below

Reply:
Do you mean like this? 
![image](https://github.com/user-attachments/assets/a2986377-d7f2-4681-a3f1-e9609e31859c)
It doesn't work:
![image](https://github.com/user-attachments/assets/e9ca49f4-2aad-492c-b872-f81b25e82aaf)
I also tried .value(), .values, and .values()

Reply:
Looks like the .value was deprecated in version 2.1, so you can ignore those comments. Anyways once we implement the ability to load a portion of the dataset the [:] notation would be the appropriate choice.